lib/harvesting/models/project_task_assignment.rb

```diff
@@ -0,0 +1,24 @@
+module Harvesting
+  module Models
+    class ProjectTaskAssignment < Base
+      attributed :id,
+                 :project,
```

Comment:
@mscottford I think `project` could use the `modeled` helper method instead of the `attributed` method.

Reply:
And a HarvestRecord base class...

Reply:
Looks like this class is actually duplicates the functionality in `Harvesting::Models::TaskAssginment`. So I'm just going to remove it.

Reply:
But I do think the naming is better, so I'll rename the existing one.

Reply:
Done.